libs/libarchfpga/src/read_xml_arch_file.cpp

```diff
@@ -4217,3 +4385,44 @@ e_side string_to_side(std::string side_str) {
     }
     return side;
 }
+
+static t_type_descriptor* get_correspondent_tile(t_type_descriptor ** Types, int NumTypes, const char *type_name) {
```

Comment:
correspondent? Did you mean corresponding?

Reply:
Done.